utilities/rpp-unittests/HIP_NEW/Tensor_hip_pkd3.cpp

```diff
@@ -978,6 +993,107 @@ int main(int argc, char **argv)
 
         break;
     }
+    case 82:
+    {
+        test_case_name = "ricap";
+        uint no_of_crops = 4;
+
+        RpptROI *cropregion = (RpptROI *) calloc(no_of_crops, sizeof(RpptROI));
+        RpptROI *d_cropregion;
+        hipMalloc(&d_cropregion, no_of_crops * sizeof(RpptROI));
+        Rpp32u initial_permute_array[images];
+        Rpp32u permuted_array1[images];
+        Rpp32u permuted_array2[images];
+        Rpp32u permuted_array3[images];
+        Rpp32u permuted_array4[images];
+        Rpp32u permuted_array[images * 4];
+
+        uint src_width = roiTensorPtrSrc[0].xywhROI.roiWidth;
+        uint src_height = roiTensorPtrSrc[0].xywhROI.roiHeight;
+
+        cropregion[0].xywhROI.xy.x =  floor(.2 * src_width);
+        cropregion[0].xywhROI.xy.y =  floor(.3 * src_height);
+        cropregion[0].xywhROI.roiWidth = floor(.75 * src_width);
+        cropregion[0].xywhROI.roiHeight = floor(.25 * src_height);
+
+        cropregion[1].xywhROI.xy.x  = floor(.1 * src_width);
+        cropregion[1].xywhROI.xy.y = floor(.4 * src_height);
+        cropregion[1].xywhROI.roiWidth = floor(.25 * src_width);
+        cropregion[1].xywhROI.roiHeight = floor(.25* src_height);
+
+        cropregion[2].xywhROI.xy.x  = floor(.4 * src_width);
+        cropregion[2].xywhROI.xy.y = floor(.15 * src_height);
+        cropregion[2].xywhROI.roiWidth = floor(.75 * src_width);
+        cropregion[2].xywhROI.roiHeight = floor(.75* src_height);
+
+        cropregion[3].xywhROI.xy.x  = floor(.2 * src_width);
+        cropregion[3].xywhROI.xy.y = floor(.2 * src_height);
+        cropregion[3].xywhROI.roiWidth = floor(.25 * src_width);
+        cropregion[3].xywhROI.roiHeight = floor(.75* src_height);
+
+        for (uint i = 0; i < images; i++ )
+        {
+            initial_permute_array[i] = i;
+        }
+        randomize(initial_permute_array, images);
+        memcpy(permuted_array1, initial_permute_array, images * sizeof(Rpp32u));
+        randomize(initial_permute_array, images);
+        memcpy(permuted_array2, initial_permute_array, images * sizeof(Rpp32u));
+        randomize(initial_permute_array, images);
+        memcpy(permuted_array3, initial_permute_array, images * sizeof(Rpp32u));
+        randomize(initial_permute_array, images);
+        memcpy(permuted_array4, initial_permute_array, images * sizeof(Rpp32u));
+
+        for(int i = 0; i < images; i++)
+        {
+            uint idx =  i * 4;
+            permuted_array[idx++] = permuted_array1[i];
+            permuted_array[idx++] = permuted_array2[i];
+            permuted_array[idx++] = permuted_array3[i];
+            permuted_array[idx++] = permuted_array4[i];
+        }
+
+        // Change RpptRoiType for ltrbROI override sample
```

Comment:
LTRB sample needs to be added and tested too like the host. Similar formatting - https://github.com/r-abishek/rpp/blob/ar/ricap_tensor/utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp#L889